podcasts/Credentials/replace_secrets.rb

```diff
@@ -53,14 +53,21 @@ def load(secrets_path)
 ## located at a given path.
 ##
 def process(template_path, secrets_path)
-  secrets = load(secrets_path)
-  template = File.open(template_path, 'r')
+  begin
+    secrets = load(secrets_path)
+    template = File.open(template_path, 'r')
 
-  template.each_line do |line|
-    puts line % secrets
+    template.each_line do |line|
+      puts line % secrets
+    end
+    template.close
+  rescue => exception
+    STDERR.puts("\n🚨🚨 Failed to generate credentials file from template: " + File.basename(template_path) +  " 🚨🚨")
+    STDERR.puts("\n-> Exception: " + exception.message)
+    STDERR.puts("-> Reason: Secrets are most likely out of date.")
+    STDERR.puts("-> Solution: Run: bundle exec fastlane run configure_apply\n\n")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/8f9153702e19dd5069e7d905af01d0f543fc0bb3/podcasts/Credentials/replace_secrets.rb#L68"></span>Use <code>warn</code> instead of <code>STDERR.puts</code> to allow such output to be disabled.</td>
    </tr>
  </tbody>
</table>